dll/cpl/sysdm/environment.c

```diff
@@ -472,7 +479,238 @@ OnDeleteVariable(HWND hwndDlg,
     }
 }
 
+static VOID
+OnEnvironmentDlgResize(HWND hwndDlg,
+                       PENVIRONMENT_DIALOG_DATA DlgData,
+                       DWORD cx,
+                       DWORD cy)
+{
+    RECT rect;
+    INT Colx, y = 0;
+    HDWP hdwp = NULL;
+    HWND hItemWnd;
+
+    if ((cx != DlgData->cxOld) || (cy != DlgData->cyOld))
```

Comment:
I would suggest, in order to remove one level of indentation, to do this:
```suggestion
    if ((cx == DlgData->cxOld) && (cy == DlgData->cyOld))
        return;
```
then comes the actual resizing code.